File: Python/D6 - T/4 - reverse of the num/4.py
# Write a python program to print the reverse number of the given number.
#  for example: input = 135, ans = 531

num = int(input('Enter the number : '))
digit = 0
rev_num = 0

while(num != 0):
    digit = num % 10
    rev_num = (rev_num * 10) + digit
    num = num // 10
    
    

print(rev_num)

File: C/D11/4/4.cpp
// Write a C program to count number of digits in a given number using function 

#include <stdio.h>
void count_digit();

int main(){
    count_digit();
}

void count_digit(){
    int num, digit = 0, count = 0;
    printf("Enter number : ");
    scanf("%d", &num);
    
    while(num != 0){
        digit = num % 10;
        count = count + 1;
        num = num / 10;
    }

    printf("Number %d digits.", count);
}